# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kde-i18n-de/hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./docs/kdeedu/kturtle/using-kturtle.docbook.orig	2005-03-11 23:04:55.008715448 -0300
+++ ./docs/kdeedu/kturtle/using-kturtle.docbook	2005-03-11 23:06:08.570532352 -0300
@@ -1011,7 +1011,7 @@
 ></term>
       <listitem
 ><para
->This is a very useful function: it provides help on the code where the cursor in the code editor is at. So, &eg;, you have used the <userinput
+>This is a very useful function: it provides help on the code where the cursor in the code editor is at. So, &zb;, you have used the <userinput
 >print</userinput
 > command in your code, and you want to read and to know what the handbook says on this command. You just move your cursor so it is in the <userinput
 >print</userinput
